components/husky_a300/husky_a300_batteries.mdx

```diff
@@ -202,4 +202,10 @@ We will configure the battery software addresses and CAN address before shipping
 
   :::
 
+  :::note When Replacing A Battery
+
+  When replacing a battery, only use an Inventus S-24V20-U1 battery.
```

Comment:
The Critical Items section lists the manufacturer as CPR, with the item number 025047. I think the CPR item is more relevant, since the battery's CAN settings need to be configured first.

Reply:
I've updated to use the CPR item here, but kept Inventus in parentheses as I believe that is the specific language requested by Nemko.